src/installer/step/manager.rs

```diff
@@ -0,0 +1,461 @@
+use bitcoin::util::bip32::ExtendedPubKey;
+use iced::{button::State as Button, scrollable, Element};
+use miniscript::DescriptorPublicKey;
+use revault_tx::scripts::{DepositDescriptor, UnvaultDescriptor};
+use std::str::FromStr;
+
+use crate::{
+    installer::{
+        message::{self, Message},
+        step::{
+            common::{CosignerKey, ParticipantXpub},
+            Context, Step,
+        },
+        view,
+    },
+    revaultd::config,
+};
+
+pub struct DefineStakeholderXpubs {
+    stakeholder_xpubs: Vec<ParticipantXpub>,
+    add_xpub_button: Button,
+    scroll: scrollable::State,
+    previous_button: Button,
+    save_button: Button,
+}
+
+impl DefineStakeholderXpubs {
+    pub fn new() -> Self {
```

Comment:
cargo clippy suggests to use Default instead of new:
```
warning: you should consider adding a `Default` implementation for `installer::step::manager::DefineStakeholderXpubs`
  --> src/installer/step/manager.rs:28:5
   |
28 | /     pub fn new() -> Self {
29 | |         Self {
30 | |             add_xpub_button: Button::new(),
31 | |             stakeholder_xpubs: Vec::new(),
...  |
35 | |         }
36 | |     }
   | |_____^
   |
   = note: `#[warn(clippy::new_without_default)]` on by default
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default
help: try this
   |
27 | impl Default for installer::step::manager::DefineStakeholderXpubs {
28 |     fn default() -> Self {
29 |         Self::new()
30 |     }
31 | }
   |
   ```
   https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default
   This holds for all the structs with a `new()` without `default()`